<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QEV1T8U" ignored="0" name="BankingApp Test" tests="5" failures="3" timestamp="24 Jul 2021 12:02:46 GMT" time="90.918" errors="0">
  <testcase name="loginTest" time="4.348" classname="com.inetbanking.testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:38)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginDDT" time="13.485" classname="com.inetbanking.testCases.Tc_LoginDDT_002"/>
  <testcase name="loginDDT" time="6.527" classname="com.inetbanking.testCases.Tc_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.Tc_LoginDDT_002.loginDDT(Tc_LoginDDT_002.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.735" classname="com.inetbanking.testCases.Tc_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.Tc_LoginDDT_002.loginDDT(Tc_LoginDDT_002.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="addNewCustomer" time="18.402" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- BankingApp Test -->
